### PR TITLE
i18n CNW hub/additional category names

### PR DESCRIPTION
**Pull request by vforge**

### CODE REVIEW

File: extensions/wikia/CreateNewWiki/CreateNewWikiController.class.php [hunk 1]
```diff
@@ -120,21 +120,60 @@ private function setupVerticalsAndCategories() {
...
+		 */
+		$this->categoriesSets = [];
 		foreach($categoriesSetsOrder as $setId => $categoriesOrder) {
-			$categoriesSet = array();
-			foreach($categoriesOrder as $categoryId) {
-				$categoriesSet[] = $allCategories[$categoryId];
+			$categoriesSet = [];
+			foreach($categoriesOrder as $categoryId => $categoryData) {
+				$categoryData = $allCategories[$categoryId];
+				$categoryData['name'] = wfMessage( 'oasis-label-category-id-' . $categoryData['id'] )->text();
```
Comment (bognix) [inline extensions/wikia/CreateNewWiki/CreateNewWikiController.class.php]: https://one.wikia-inc.com/wiki/Engineering/Onboarding/Internationalization/Technical/Advanced#Outputting_messages_to_HTML - I'm not sure if escaped() in necessary but it's going to be placed in HTML

**Comment by vforge** [comment]
Pinging @lgarczewski too :)

**Comment by bognix** [comment]
LGTM

**Comment by hakubo** [comment]
LGTM thansk

**Pull request merged**